db/migrate/20201127094738_create_placement_need_profiles.rb

```diff
@@ -0,0 +1,23 @@
+class CreatePlacementNeedProfiles < ActiveRecord::Migration[6.0]
+  def change
+    create_table :placement_need_profiles do |t|
+      t.belongs_to :child
+      t.date :placement_date
+      t.boolean :long_term, default: false, null: false
```

Comment:
With these "placement types" or "placement categories". The more we spoke about it at the end of the week, I thought about maybe needing a resource here. Because:
1) Even though the placement need for a child is of one type, the `FosterParent` will be approved for multiple. 
2) I suspect that these categories will vary between different local authorities. Remember when Luke was talking about multiple types of "emergency" for instance. 

We may decide not to implement it at this stage, but I think we may need it at some point.

Reply:
I agree, it will be a resource, we can put it on another PR